CHANGELOG.md

```diff
@@ -1,12 +1,9 @@
 # Changelog
 
 ## v1.13.1
- {release_date}
-### Upgrade requirements
-### Migrations and checks
-#### Schema migrations
-#### Data migrations
+ 2021-06-07
 ### Changes
+- Map composition schema allows new properties `hs.format.externalWFS` and `workspace`. It also has more detailed description of `style` property.
```

Comment:
- [Map composition schema](https://github.com/LayerManager/layman/blob/master/src/layman/map/schema.draft-07.json) allows new properties `hs.format.externalWFS` and `workspace` (#382). The `style` property can be specified using a string in SLD format, URL to SLD file or JSON object (#385).